Makefile

```diff
@@ -2,12 +2,7 @@
 default: build-latest run-dev
 
 build-base:
-	base/hacks/kilda-bins.download.sh
 	base/hacks/shorm.requirements.download.sh
```

Comment:
It could be great to rename shorm.requirements.download.sh to storm.requirements.download.sh.